core/src/main/java/com/linecorp/armeria/internal/PathMappingUtil.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2018 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.internal;
+
+import java.util.Map;
+import java.util.regex.Pattern;
+
+import com.linecorp.armeria.server.PathMapping;
+
+public final class PathMappingUtil {
+
+    //FIXME: Duplicate of {@link DefaultPathMapping#VALID_PATTERN}
+    private static final Pattern VALID_DEFAULT_PATTERN = Pattern
+            .compile("(?:/[^/{}:]+|/:[^/{}]+|/\\{[^/{}]+})+/?");
+
+    //FIXME: Currently unused
+    public static final String EXACT_PREFIX = "exact:";
+    public static final String PREFIX_PREFIX = "prefix:";
+    public static final String GLOB_PREFIX = "glob:";
+    public static final String REGEX_PREFIX = "regex:";
```

Comment:
Could you remove unused stuff? You could create an issue if you are interested in adding support for these features.